services/net/contentmigration/migrators/ClipMigrator.cs

```diff
@@ -73,6 +73,17 @@ public ClipMigrator(IApiService api, IOptionsSnapshot<MigratorOptions> migratorO
             content.UpdatedOn = newsItem.UpdatedOn != DateTime.MinValue ? newsItem.UpdatedOn.Value.ToUniversalTime() : null;
         }
 
+        if (newsItem.Tones != null) {
+            // TODO: replace the USER_RSN value on UserIdentifier with something that can be mapped by the Content Service to an MMIA user
+            // TODO: remove UserRSN filter once user can be mapped
+            content.TonePools = newsItem.Tones.Where(t => t.UserRSN == 0)
+                .Select(t => new Kafka.Models.TonePool { Value = t.ToneValue, UserIdentifier = t.UserRSN.ToString() });
```

Comment:
It'll have to be email or username, or some combination regrettably.

Reply:
I added a story to take care of this lookup/mapping on both sides
https://apps.itsm.gov.bc.ca/jira/browse/TNO-1436